common/lib/xmodule/xmodule/services.py

```diff
@@ -0,0 +1,19 @@
+"""
+Module contains various XModule/XBlock services
+"""
+from django.conf import settings
+
+class SettingsService(object):
+    """
+    Allow XBlocks to access the server settings
```

Comment:
Nit: Would be more clear to say "Allows server-wide configuration of XBlocks on a per-type basis"